docs/tutorials/iers_a.rst

```diff
@@ -0,0 +1,70 @@
+:orphan:
+
+.. doctest-skip-all
+
+**********************************
+Using and updating IERS Bulletin A
+**********************************
+
+Contents
+========
+
+* :ref:`iers_a-Background`
+* :ref:`iers_a-What_are_IERS_Bulletins_A_and_B`
+* :ref:`iers_a-How_do_I_get_IERS_Bulletin_A_for_astroplan?`
+
+.. _iers_a-Background:
+
+Background
+==========
+
+Keeping accurate, consistent time systems is rough. We intuitively want
+time to be described by a continuous cyclic clock like the clock on your
+wall or the calendar, but the Earth's rate of rotation varies measurably
+on human timescales. As a result, the time measured by an atomic clock and
+the time that you would infer from measuring how long it has been since the
+last solar noon are constantly becoming offset from one another with
+stochastic changes in Earth's moment of inertia and slowly acting tidal forces.
+
+For this reason, there is a time system that keeps track of seconds like an
+atomic clock, UTC, one that matches up with the Earth's rotation, UT1. The
+difference between UT1 and UTC is constantly changing as the Earth's rotation
+changes and as leap seconds get added to UTC to compensate.
+
+In order to accurately predict the apparent position of a celestial object from
+the Earth, for example in altitude and azimuth coordinates, one needs to know
+the orientation of the Earth at any point in time, and since the Earth's
+rotation does not change in a predictable way, we must rely on observations
+of the Earth's orientation as a function of time to accurately calculate
+UT1-UTC.
```

Comment:
I think this description in "Background" is excellent ... keeps it very simple, but addresses the core point.
Maybe add your plot of UT1-UTC here, or link to your blog post?